test/base_maths.jl

```diff
@@ -111,5 +111,16 @@ using Test
             @test D * b1 ≈ D * Matrix(b1)
             @test_throws DimensionMismatch D′ * b1
         end
+
+        @testset "Non-Square BlockDiagonal * Non-Square BlockDiagonal" begin
+    		b4 = BlockDiagonal([ones(2, 4), 2 * ones(3, 2)])
```

Comment:
Indentation off :)

Reply:
I think the problem is that my IDE settings do not match yours. 😄